### DISCUSSION TITLE
Sudoku Studio V3.4.0 Preview

### DISCUSSION THREAD

**Discussion by kyoyama-kazusa** [SunnieShine/Sudoku#701] [General]
Please remove configuration files in your device if you had used this program before this release.

---

**Full Changelog**: https://github.com/SunnieShine/Sudoku/compare/v3.3.2...v3.4.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/SunnieShine/Sudoku/releases/tag/v3.4.0-beta'>Sudoku Studio V3.4.0</a>.</em>

**Comment by weisilisail** [comment]
向神，能否增加一个可以调整九宫格大小的框呢，笔记本屏幕小显示不完全，感谢感谢。

**Reply by kyoyama-kazusa** [reply]
#706
